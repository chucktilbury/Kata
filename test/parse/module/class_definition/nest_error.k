;-------------------------------------------------------------------------------
; Verify that a nested class produces an error
;-------------------------------------------------------------------------------

class name {

    class name {

    }
}
