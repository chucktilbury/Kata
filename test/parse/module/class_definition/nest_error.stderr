Syntax: /home/chuck/Src/Kata/test/parse/module/class_definition/nest_error.k: 7: 10: expected class item or a '}' but got class
Syntax: /home/chuck/Src/Kata/test/parse/module/class_definition/nest_error.k: 7: 10: expected module item or end of file but got class
Syntax: /home/chuck/Src/Kata/test/parse/module/class_definition/nest_error.k: 7: 10: module cannot be empty
